include/boost/url/impl/url_view.ipp

```diff
@@ -818,4 +822,46 @@ operator<<(
 } // urls
 } // boost
 
+//----------------------------------------------------------
+//
+// std::hash specialization
+//
+//----------------------------------------------------------
+
+std::size_t
+std::hash<::boost::urls::url_view>::operator()(
+    ::boost::urls::url_view const& u) const noexcept
+{
+#if BOOST_URL_ARCH == 64
+    std::uint64_t const prime = 0x100000001B3ULL;
```

Comment:
These constants came from boost.json, but I don't know the procedure for coming up with new constants.

Reply:
You don't need new constants, those are fine (I think). It would have to be analyzed mathematically. Out of scope for now, until someone complains.